index.html

```diff
@@ -0,0 +1,276 @@
+          </button>
+        </div>
+      </section>
+      <section class="speakers">
+        <div class="container">
+          <h2 class="speakers__headline small-underline">Featured Speakers</h2>
+          <div
+            id="speakers__content"
+            class="speakers__content"
+            data-expanded="false"
+          ></div>
+        </div>
+      </section>
+      <section class="partners">
+        <div class="container">
+          <h2 class="partners__headline small-underline">Partners</h2>
+          <ul class="partners__cards-wrapper">
+            <li class="partners__logo">
+              <a href="#"
+                ><img
+                  class="partners__logo__img"
+                  src="assets/partner-logos/zulu.svg"
+                  alt=""
+              /></a>
+            </li>
+            <li class="partners__logo">
+              <a href="#"
+                ><img
+                  class="partners__logo__img"
+                  src="assets/partner-logos/buzzzer.svg"
+                  alt=""
+              /></a>
+            </li>
+            <li class="partners__logo">
+              <a href="#"
+                ><img
+                  class="partners__logo__img"
+                  src="assets/partner-logos/axon.svg"
+                  alt=""
+              /></a>
+            </li>
+            <li class="partners__logo">
+              <a href="#"
+                ><img
+                  class="partners__logo__img"
+                  src="assets/partner-logos/lotor.svg"
+                  alt=""
+              /></a>
+            </li>
+            <li class="partners__logo">
+              <a href="#"
+                ><img
+                  class="partners__logo__img"
+                  src="assets/partner-logos/muda.svg"
+                  alt=""
+              /></a>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <div id="overlay" class="overlay"></div>
+    </main>
+    <footer class="footer">
+      <div class="container">
+        <div class="footer__wrapper">
+          <a href="/" class="footer__logo__wrapper">
+            <img class="footer__logo" src="assets/icons/logo-lg.svg" alt="" />
+          </a>
+          <div>
+            <p class="footer__copyright">
+              &#169; 2023 African Education Summit. Some rights reserved.
+            </p>
+            <p class="footer__disclaimer">
+              All eniities represented on this website, including the partners,
+              speakers and program, are all fictional and were created for the
+              purpose of a web development project. The source code for this
+              website can be found
+              <a
+                class="footer__github-link"
+                href="https://github.com/henokkhm/microverse-module-1-capstone-project"
+              >
+                here </a
+              >.
+            </p>
+          </div>
+        </div>
+      </div>
+    </footer>
```

Comment:
- [ ]  Kindly make sure to remove the footer and Partner section from the Home page in the mobile version so it looks identical to the provided design

|Your website|Provide design|
|-|-|
|![Screenshot_20230529_234214](https://github.com/henokkhm/microverse-module-1-capstone-project/assets/25227022/4757470e-d072-443b-a23a-cd1212a88867)|![Screenshot_20230529_234237](https://github.com/henokkhm/microverse-module-1-capstone-project/assets/25227022/8a527948-56ef-4f13-8d15-16a8ee227eab)|